src/components/common/Input.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Input = (props) => {
+  const {
+    type,
+    name,
+    value,
+    className,
+    placeholder,
+    handleChange,
+    icon,
+    errors
+  } = props;
+  return (
+    <div className="field">
+      <div className="ui left icon input">
+        <input
+          type={type}
+          name={name}
+          className={className}
+          placeholder={placeholder}
+          value={value}
+          onChange={handleChange}
+          errors={errors}
+          required
+        />
+        <i className={icon} />
+      </div>
+      {errors && (
+        <div className="ui pointing red basic label">
+          {errors}
+        </div>
+      )}
+    </div>
+  );
+};
+
+Input.defaultProps = {
+  className: "",
+  type: "text",
+  placeholder: "Enter input"
+};
+
+Input.propTypes = {
+  type: PropTypes.string,
+  name: PropTypes.string,
+  value: PropTypes.string,
```

Comment:
This should be required

Reply:
I thought you said I should not make it required, that  I should set defaultValues instead